examples/container-definition/outputs.tf

```diff
@@ -12,9 +12,16 @@ output "container_definition_json" {
   value       = module.ecs_container_definition.container_definition_json
 }
 
-resource "local_file" "container_definition_json" {
-  content  = module.ecs_container_definition.container_definition_json
-  filename = "${path.module}/definition.json"
+# Need the output pretty-printed and sorted for comparison
+resource "null_resource" "container_definition_json" {
```

Comment:
the `local_file` doesn't output in pretty-printed JSON so switched to this hack to ensure its formatted and sorted to be able to compare diffs

Reply:
Why do we need `definition.json` other than for debugging? Consider making it conditional, so users can turn it off if needed.